index.html

```diff
@@ -0,0 +1,28 @@
+<html>
+  <head>
+    <link rel="stylesheet" href="style.css">
+    <script src="index.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.9.3/Chart.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/randomcolor/0.5.4/randomColor.min.js"></script>
+  </head>
+  <body>
+    <h1>Bill Split Calculator</h1>
+
+    <section class="form">
+      <article class="col">
+        <form>
+          <label for="total">Bill Total: $</label>
+          <input id="total" name="total" type="number" min="0" step="0.01" />
+          <br />
+          <label for="guests">Number of Guests:</label>
+          <input id="guests" name="guests" type="number" min="1" step="1" />
+        </form>
+      </article>
+
+      <article class="col graph-container">
```

Comment:
You're not using `graph-container` class and `.col` `article` isn't differently styled from any other `article` elements on page, so you can just get rid of these CSS classes and use `article {...}` in your stylesheet. Also, though `col` does have semantic value here by implying you want to have columns, perhaps it wouldn't be the case for a mobile screen width.